coldFilmSearchParser.js

```diff
@@ -0,0 +1,70 @@
+const htmlparser = require('htmlparser2');
+
+function coldFilmSearchParser(str){
+    let parseLink = false;
```

Comment:
is this variable will be `bool` for all time during function execution?
if it's true rename it
now it's sounds like in this variable will be `sting` or something like this, but not `bool`